samples/commit-dir.yaml

```diff
@@ -0,0 +1,121 @@
+# Prerequisites:
+# - Copy to a new GitHub Actions workflow file in the .github/workflows directory.
+# - Update the environment variables for your setup.
+# - Consider writing results to an alternative GitHub repository if security is a concern.
+
+# Overall workflow:
+# 1. List experiments (read from App Configuration file or manually specify a single feature flag).
+# 2. Analyze each experiment and save the results summary to a Markdown file.
+# 3. Commit the analysis results to a directory in the GitHub repository.
+
+name: Analyze Experiments
+
+on:
+  # trigger automatically (daily at 12:30 AM UTC)
+  schedule:
+    - cron: "30 0 * * *"
+  # trigger manually (via GitHub Actions UI)
+  workflow_dispatch:
+    inputs:
+      app-configuration-feature-flag:
+        description: "The App Configuration feature flag. By default, analyze all running experiments."
+        required: false
+        type: string
+
+# Update for your setup
+env:
+  ANALYSIS_DIR: .config/analysis
+  APP_CONFIGURATION_FILE: .config/feature-flags.json
```

Comment:
Can we support glob here ?

Reply:
Yes, I've just added multi-file support via the [jq inputs feature](https://jqlang.github.io/jq/manual/#inputs). This supports a list of filenames here or using bash wildcard expansion.